src/identity/Identity.ts

```diff
@@ -28,9 +28,43 @@ type BoxPublicKey =
 
 export default class Identity extends PublicIdentity {
   private static ADDITIONAL_ENTROPY_FOR_HASHING = new Uint8Array([1, 2, 3])
+  /**
+   * [STATIC] Generates Mnemonic phrase used to create identities from phrase seed.
+   * @returns `generate`
```

Comment:
Fix: see what `generateMnemonic` returns in https://kiltprotocol.github.io/kilt-workshop-101/#/identity

Reply:
I am returning the generated mnemonic. How do you mean?